tools/clang/unittests/DxrFallback/test_DxrFallback.cpp

```diff
@@ -161,8 +161,8 @@ class Tester {
   }
 
 protected:
-  DxcDllSupport m_dxcSupport;
-  DxcDllSupport m_dxrFallbackSupport;
+  SpecificDllLoader m_dxcSupport;
+  SpecificDllLoader m_dxrFallbackSupport;
```

Comment:
Would it be clearer if `m_dxcSupport` were a `DxCompilerDllLoader` here? `m_dxrFallbackSupport` being a different `SpecificDllLoader` makes sense though.